xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/viewers/attachments.js

```diff
@@ -218,29 +218,46 @@ require(['jquery', 'xwiki-events-bridge'], function($) {
       }
     })
   });
+  $(document).on("xwiki:livetable:loadingComplete", function(event, data) {
+    if (data.tableId == 'docAttachments') {
+      $(document).on('click', '#docAttachments .actiondelete', function(event) {
+        event.preventDefault();
+        $('#deleteAttachment').data('relatedTarget', event.currentTarget);
```

Comment:
Here you don't wrap ``event.currentTarget`` with jQuery. So you need to decide: wrap everywhere or nowhere.